WordPress/Classes/ViewRelated/Reader/ReaderFollowedSitesViewController.swift

```diff
@@ -0,0 +1,332 @@
+import Foundation
+import WordPressShared
+import SVProgressHUD
+
+///
+///
+class ReaderFollowedSitesViewController: UIViewController
+{
+    @IBOutlet var searchBar: UISearchBar!
+
+    private var refreshControl: UIRefreshControl!
+    private var isSyncing: Bool = false
+    private var tableView: UITableView!
+    private var tableViewHandler: WPTableViewHandler!
+    private var tableViewController: UITableViewController!
+
+    private let cellIdentifier = "CellIdentifier"
+
+    lazy var noResultsView: WPNoResultsView = {
+        let title = NSLocalizedString("No Sites", comment: "Title of a message explaining that the user is not currently following any blogs in their reader.")
+        let message = NSLocalizedString("You are not following any sites yet. Why not follow one now?", comment: "A suggestion to the user that they try following a site in their reader.")
+        return WPNoResultsView(title: title, message: message, accessoryView: nil, buttonTitle: nil)
+    }()
+
+
+    /// Convenience method for instantiating an instance of ReaderFollowedSitesViewController
+    ///
+    /// - Returns: An instance of the controller
+    ///
+    class func controller() -> ReaderFollowedSitesViewController {
+        let storyboard = UIStoryboard(name: "Reader", bundle: NSBundle.mainBundle())
+        let controller = storyboard.instantiateViewControllerWithIdentifier("ReaderFollowedSitesViewController") as! ReaderFollowedSitesViewController
+        return controller
+    }
+
+
+    // MARK: - LifeCycle Methods
+
+
+    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
+        tableViewController = segue.destinationViewController as? UITableViewController
```

Comment:
You may want to set an identifier on the segue and check it here, for future proofing?